jac-uk/admin
# move info to tooltip

## What's included?
UI change: https://github.com/jac-uk/admin/pull/2605#issuecomment-2497686811

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
- Go to the preview link: https://jac-admin-develop--pr2616-feat-admin-2579-add-vk06x3ei.web.app/exercise/Biyjd07Xz2usL9yXjtjV/dashboard
- The tooptip should be added:
<img width="657" alt="Screenshot 2024-11-26 at 12 38 34" src="https://github.com/user-attachments/assets/5e8119ea-513b-4d36-b1dc-874057d084e1">

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Include screen grabs, video demo, notes etc.

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required
- Permissions have been added / updated. Details:

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
